compiler/src/iree/compiler/InputConversion/StableHLO/StableHLOToLinalgReduce.cpp

```diff
@@ -0,0 +1,692 @@
+// Copyright 2019 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
nit: bit weird that a "StableHLO" file is dated 2019, before StableHLO was even created

The year should be whatever year _this code_ was first written and should be kept unchanged as the file is moved or forked. If new code is substantially different, it can get a new year...

Reply:
This is the creation year of the mhlo equivalent of this file